generated/beats/fields.ecs.yml

```diff
@@ -2505,877 +3320,4822 @@
       default_field: false
-    - name: parent.title
+    - name: process.hash.md5
       level: extended
       type: keyword
       ignore_above: 1024
-      multi_fields:
-      - name: text
-        type: text
-        norms: false
-      description: 'Process title.
-
-        The proctitle, some times the same as process name. Can also be different:
-        for example a browser setting its title to the web page currently opened.'
+      description: MD5 hash.
       default_field: false
-    - name: parent.uptime
+    - name: process.hash.sha1
       level: extended
-      type: long
-      description: Seconds the process has been up.
-      example: 1325
+      type: keyword
+      ignore_above: 1024
+      description: SHA1 hash.
       default_field: false
-    - name: parent.working_directory
+    - name: process.hash.sha256
       level: extended
       type: keyword
       ignore_above: 1024
-      multi_fields:
-      - name: text
-        type: text
-        norms: false
-      description: The working directory of the process.
-      example: /home/alice
+      description: SHA256 hash.
       default_field: false
-    - name: pgid
-      level: extended
-      type: long
-      format: string
-      description: Identifier of the group of processes the process belongs to.
-    - name: pid
-      level: core
-      type: long
-      format: string
-      description: Process id.
-      example: 4242
-    - name: ppid
-      level: extended
-      type: long
-      format: string
-      description: Parent process' pid.
-      example: 4241
-    - name: start
-      level: extended
-      type: date
-      description: The time the process started.
-      example: '2016-05-23T08:05:34.853Z'
-    - name: thread.id
-      level: extended
-      type: long
-      format: string
-      description: Thread ID.
-      example: 4242
-    - name: thread.name
+    - name: process.hash.sha512
       level: extended
       type: keyword
       ignore_above: 1024
-      description: Thread name.
-      example: thread-0
-    - name: title
-      level: extended
+      description: SHA512 hash.
+      default_field: false
+    - name: process.hash_matched_module
+      level: custom
+      type: boolean
+      description: TODO
+      default_field: false
+    - name: process.imphash
+      level: custom
       type: keyword
       ignore_above: 1024
-      multi_fields:
-      - name: text
-        type: text
-        norms: false
-        default_field: false
-      description: 'Process title.
-
-        The proctitle, some times the same as process name. Can also be different:
-        for example a browser setting its title to the web page currently opened.'
-    - name: uptime
-      level: extended
-      type: long
-      description: Seconds the process has been up.
-      example: 1325
-    - name: working_directory
-      level: extended
+      description: TODO
+      default_field: false
+    - name: process.is_sensor
```

Comment:
Consider changing to `is_endpoint`. Sensor is an SMP term not an Elastic Endpoint term.